src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -145,6 +149,7 @@ public void close() throws IOException {
     }
 
     public SeekableByteChannelPrefetcher(SeekableByteChannel chan, int bufSize) throws IOException {
+        this.prefetcherIndex = (prefetcherCount++);
```

Comment:
Yup, it's [no problem](http://docs.oracle.com/javase/6/docs/api/java/lang/Thread.html):

> More than one thread can have the same name